pkg/controller/migplan/pvlist.go

```diff
@@ -0,0 +1,106 @@
+package migplan
+
+import (
+	"context"
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	migref "github.com/fusor/mig-controller/pkg/reference"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type PvTable map[types.NamespacedName]core.PersistentVolume
```

Comment:
Thoughts on PvMap vs PvTable here? Seems like a map

Reply:
Agreed. `Map` naming and semantics is better.